CHANGES/9685.breaking.rst

```diff
@@ -0,0 +1 @@
+The ``FlowControlDataQueue`` has been replaced with the ``WebSocketDataQueue`` -- by :user:`bdraco`.
```

Comment:
This wasn't broken in any public stable release so no bugfix changelog message.  

It likely wouldn't warrant a changelog message if it wasn't for `FlowControlDataQueue` was available for import in `aiohttp` top level namespace, although it would be a bit unexpected that someone would use it externally, its technically a breaking change.